packages/elements/src/calendar/types.ts

```diff
@@ -62,4 +62,5 @@ export type CalendarCell = {
 // public API
 export type BeforeCellRenderEvent = CustomEvent<{
   cell: CalendarCell;
+  calendar: HTMLElement;
```

Comment:
Let's improve type definition.
```suggestion
  calendar: Calendar;
```

Reply:
Done